### ISSUE TITLE
Empty settlement window query causes UI error

### ISSUE THREAD

**Issue by partiallyordered** [mojaloop/finance-portal-v2-ui#27]
![Selection_234](https://user-images.githubusercontent.com/6493470/129546050-af52dab1-b12a-42f5-b2d9-a80b75839f34.png)

Root cause: calling central settlement `GET /v2/settlementWindows` with no query parameters returns:
```json
{"errorInformation":{"errorCode":"3100","errorDescription":"Generic validation error - Use at least one parameter: participantId, state, fromDateTime, toDateTime, currency"}}
```
which propagates through the UI

**Comment by partiallyordered** [comment]
See also #22 (is that the same?)

**Comment by partiallyordered** [comment]
Resolved in #59 by using default from and end dates like so:
https://github.com/mojaloop/finance-portal-v2-ui/pull/59/files#diff-1f50adeffe650626116d19665f33e349921aa4da201e36b0499ab33ff4aba979R48-R52
